### PR TITLE
#5537 [Manager] Bump Material Table to fix React warnings

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
@eechih After upgrading to material table v1.68.0, I'm no longer seeing these react warnings. Could you help me to verify this locally with your machine?

If you're still seeing it after `yarn setup`, you could try to delete client's node_modules and run `yarn setup` again 

Much appreciated 🙏

File: ohara-manager/client/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "jointjs": "3.2.0",
     "jquery": "3.5.1",
     "localforage": "1.8.1",
-    "material-table": "1.67.1",
+    "material-table": "1.68.0",
```
Comment (konekoya) [inline ohara-manager/client/package.json:65]: Looks like the version bump does fix the issue, but I will see if this breaks any of our tests or not

**Comment by eechih** [comment]
> After upgrading to material table v1.68.0, I'm no longer seeing these react warnings. Could you help me to verify this locally with your machine?

LGTM, thanks @konekoya

**Review by eechih — APPROVED** [review]

**Comment by konekoya** [comment]
retry manger-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@eechih Unfortunately, this version introduces another bug that crashes the entire table when adding a new row (in our schema table) Many users are encountering this a well: https://github.com/mbrn/material-table/issues/2269

And sadly, the latest release also has a bug: https://github.com/mbrn/material-table/issues/2400

The best bet for us now is probably just still at v1.67.1 which still has the warnings but at least all features work 😅

**Comment by eechih** [comment]
> The best bet for us now is probably just sticking with v1.67.1 and despite it has the warnings, at least all features work in this version 😅

OK :)

**Pull request closed without merging**